termio.go

```diff
@@ -51,3 +51,13 @@ func getWindowDimensions() (int, int) {
 func clear() {
 	fmt.Print("\033[H\033[2J")
 }
+
+// hideCursor makes the cursor disappear
+func hideCursor() {
+	os.Stdout.WriteString(fmt.Sprintf("\x1B[?25l"))
```

Comment:
What's the benefit of doing the WriteString here ? As opposed to fmt.Printf(...

Reply:
i've gotten paranoid of fmt.Printf generating double bytes for unicode, but i think this is unnecessary, i'll swap it to a printf and give it a go